src/System.Windows.Forms/src/System/Windows/Forms/ToolStrip.ToolStripAccessibleObject.cs

```diff
@@ -338,6 +412,11 @@ public override AccessibleRole Role
                 }
             }
 
+            private bool ShouldItemBeSkipped(Control hostedControl)
```

Comment:
What do you think of this name?
```suggestion
            private bool IsValidFragment(Control hostedControl)
```

Reply:
Hm, looks not good to me. I wouldn't say that some item are not valid. They just should be skipped to avoid duplications. But the are valid. I would keep it as is. Maybe there is possible another better name.